html-forms/index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Forms</title>
+  </head>
+  <body>
+    <h1>Coding Form!</h1>
+      <div>
+        <form action="index.html">
+          <p>Email: <br>
+            <input type="text" name="email" size="15" maxlength="30" />
+          </p>
+        </form>
+      </div>
+      <div>
+        <form action="index.html">
+          <p>Why is coding cool?</p>
+          <select name="reasons">
+            <option value="interesting">Because it's interesting</option>
+            <option value="fun">Because it's fun</option>
+            <option value="powerful">Because it's powerful</option>
+          </select>
+        </form>
+      </div>
+      <div>
+        <form action="index.html">
+          <p>Favorite Language (choose one)
+            <br>
+            <input type="radio" name="language" value="html"
+              checked="checked" /> HTML
+            <input type="radio" name="language" value="css" />
+              CSS
+            <input type="radio" name="language" value="javascript" />
+              JavaScript
+          </p>
```

Comment:
A `<label>` should be used for each `radio` button. Also avoid using `<br>` tags

Reply:
Hey Scott, I made all of the other changes but wanted clarification for this comment. When a <label> is used for each radio button, does that mean I change <input type=> to <label> ?

Here is my code below:
<input type="radio" name="language" value="css" />
              CSS

Thanks